lib/msal-browser/docs/v4-migration.md

```diff
@@ -161,6 +171,88 @@ See the [Configuration doc](./configuration.md#system-config-options) for more d
 
 The `onRedirectNavigate` parameter will *only be supported* from `Configuration` object going forward and is removed from `RedirectRequest` and `EndSessionRequest` objects. Please ensure to set it in msal config if you need to use it.
 
+### Consolidation of extra request parameters
+
+The following request parameters have been removed:
```

Comment:
Same as above. I would move this to PS as a note and highlight the PS with a removal notice as needed.

Reply:
I meant can we mention the new support first and then follow up with removed params. Better for readability.